### PR TITLE
AV-162483: AKO optimization setting document

### PR DESCRIPTION
**Pull request by akshayhavile**

### CODE REVIEW

File: docs/ako_optimisation.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+For example, if user wants to block syncing objects from `kube-system`, `kube-public` namespaces, user can specify those namespaces as follows:
+
+  AKOSettings:
+    .
+    .
+    blockedNamespaceList:
+      - kube-system
+      - kube-public
+
+AKO reboot will be required if value of this knob changes.
```
Comment (swathinsankaran) [inline docs/ako_optimisation.md:65]: we must this as a note to the user. 
**Note** AKO reboot will be required if the value of this knob changes.

File: docs/ako_optimisation.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+        kubernetes.io/os: linux
+        nodeselected: yes
+      name: node2
+    spec:
+      .
+      .
+      .
+```
+
+AKO will select `node2` while populating pool servers.  If value of this knob changes, AKO reboot will be required.
```
Comment (swathinsankaran) [inline docs/ako_optimisation.md:97]: We have to specify the change in ako config map requires a reboot of AKO as users might get confused with config change to node also.

Comment (akshayhavile) [reply docs/ako_optimisation.md]: addressed

File: docs/ako_optimisation.md (hunk 1 above)
Comment (akshayhavile) [reply docs/ako_optimisation.md]: addressed

**Review by swathinsankaran — APPROVED** [review]
LGTM

**Review by aaha97 — APPROVED** [review]
LGTM

**Comment by akshayhavile** [comment]
build ako

**Comment by swathinsankaran** [comment]
build ako

**Pull request merged**